### ISSUE TITLE
Version 2.4.0 fails to extract

### ISSUE THREAD

**Issue by quentez** [lingui/js-lingui#260]
**Describe the bug**
Since upgrade to version `2.4.0` on a TypeScript project, we are unable to run `lingui extract`.

```
Unknown option: .projectType. Check out http://babeljs.io/docs/usage/options/ for more information about options.
```

**To Reproduce**
- Install version `2.4.0` in a TS project.
- Run `lingui extract` on that project.

**Expected behavior**
No error.

**Additional context**

- jsLingui version ``2.4.0``
- Babel version ``7.0.0-bridge.0``
- React + Typescript.

**Comment by tricoder42** [comment]
Released in [v2.4.1](https://github.com/lingui/js-lingui/releases/tag/v2.4.1), thanks for reporting the issue!

**Comment by quentez** [comment]
Thanks for the super quick release :) !!
